charts/descheduler/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: descheduler
-version: 0.20.0
+version: 0.20.1
```

Comment:
wondering if we should bump+release a new chart version now, or just wait for the v0.21 descheduler release (which is coming Soon) and release the updated chart then?